### ISSUE TITLE
Accent in rticles packages

### ISSUE THREAD

**Issue by arturochian** [rstudio/rticles#21]
Hi, I would like to write some examples in spanish. I have some problems with accents. For example when I write "análisis" my output is like this
![image](https://cloud.githubusercontent.com/assets/4960935/5593724/a567642e-91f2-11e4-9a8f-512add97aa53.png)

I think it's related to encoding in latex (or maybe YALM?).

Thank in advance

**Comment by hadley** [comment]
Could you please provide a minimal reproducible example?

**Comment by arturochian** [comment]
@hadley  a minimal reproducible example is here
https://github.com/arturochian/repro1
PD: I'm using JSS

**Comment by arturochian** [comment]
Let see the introduction and the title

![image](https://cloud.githubusercontent.com/assets/4960935/5599095/4309dc4c-928f-11e4-9660-05a68e2c1f4e.png)

**Comment by arturochian** [comment]
And this is my sessionInfo

R version 3.1.1 (2014-07-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)

locale:
[1] LC_COLLATE=Spanish_Peru.1252  LC_CTYPE=Spanish_Peru.1252    LC_MONETARY=Spanish_Peru.1252
[4] LC_NUMERIC=C                  LC_TIME=Spanish_Peru.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] digest_0.6.4    htmltools_0.2.7 rmarkdown_0.3.9 rticles_1.0     tools_3.1.1    
[6] yaml_2.1.13

**Comment by arturochian** [comment]
@hadley  I have recently noticed this message from R. I tried with differents encoding but they don't work. Maybe the solution is related to ctex (in chinese) template creation (maybe @yihui  could also know about this behavior).

"
Not all of the characters in ~/BCRP/a/Untitled/Untitled.Rmd could be encoded using ISO8859-1. To save using a different encoding, choose "File | Save with Encoding..." from the main menu.Error in yaml::yaml.load(front_matter) : 
  Reader error: invalid leading UTF-8 octet: #AD at 367
Error in yaml::yaml.load(front_matter) : 
  Reader error: invalid leading UTF-8 octet: #AD at 340
Error in yaml::yaml.load(front_matter) : 
  Reader error: invalid leading UTF-8 octet: #AD at 340

"
